problem_builder/public/js/lms_util.js

```diff
@@ -0,0 +1,109 @@
+(function () {
```

Comment:
This can be moved to ``step.js`` but I figured that it would be better to have it on separate file, to keep every file simple enough.